### PR TITLE
Add anchor for GitHub permalink in `doc/text/news.md`

### PR DESCRIPTION
**Pull request by kachick**
This change makes useful to me 😄 How do you think?

NOTE: Replaced all of them as `PR`.  if it is `Issue`, GitHub will forwards

### CODE REVIEW

**Comment by kou** [comment]
I think so too. But it bothers me when I update `news.md`.
How about adding a script that converts `[GitHub#X]` to `[GitHub#X](...)`? Then I can run the script after I update `news.md`.

**Comment by kachick** [comment]
Sounds good! I'll try (But it might be rough script when implemented by me... 😅 )

**Comment by kachick** [comment]
Sorry for the late updating 🙇‍♂️ 

I have added script as a rake task. `rake doc:add_permalink`

https://github.com/test-unit/test-unit/pull/218/files#diff-456407efc98b454fc1490231a709f6d740adfa1707b100c07040dd7dada3398fR2587-R2602 is failing, however it looks not related to this change. 🤔

File: Rakefile [hunk 1]
```diff
@@ -55,3 +55,24 @@ end
...
 end
+
+namespace :doc do
+  task :add_permalink do
+    news_md = File.read("doc/text/news.md")
+    base_url = "https://github.com/test-unit/test-unit/pull/"
+
+    applied_typed_permalink = news_md.gsub(/(?<pre>\[GitHub#(?<ref>\d+)\])(?<post> ?\[(?<type>Report|Patch))/) do
+      if Regexp.last_match[:type] == "Report"
+        base_url = "https://github.com/test-unit/test-unit/issues/"
```
Comment (kou) [inline Rakefile:66]: If we find any `Report` type, all rest `[GitHub#XXX]`s use `issues/` not `pull/`.

BTW, GitHub accepts `issues/XXX` for pull requests. So we can always use `issues/`.

Comment (kachick) [reply Rakefile]: >If we find any Report type, all rest [GitHub#XXX]s use issues/ not pull/.

Sorry for the bug 🙇‍♂️  Fixed in https://github.com/test-unit/test-unit/pull/198/commits/03fbf37aa5c7710a917f7a569eeb911b96f38430

>So we can always use issues/.

Updated with `issues` suffix in https://github.com/test-unit/test-unit/pull/198/commits/1f3ab61ad38cfd906b6ebe47d459b12edbac2429.

File: Rakefile [hunk 2]
```diff
@@ -55,3 +55,14 @@ end
 task :test do
   ruby("test/run-test.rb")
 end
+
+namespace :doc do
+  task :add_permalink do
+    news_md = File.read("doc/text/news.md")
+    applied_permalink = news_md.gsub(/(?<pre>\[GitHub#(?<ref>\d+)\])(?!\()/) do
+      "#{Regexp.last_match[:pre]}(#{"https://github.com/test-unit/test-unit/issues/"}#{Regexp.last_match[:ref]})"
```
Comment (kou) [inline Rakefile:63]:
```suggestion
      "#{Regexp.last_match[:pre]}(https://github.com/test-unit/test-unit/issues/#{Regexp.last_match[:ref]})"
```

Comment (kachick) [reply Rakefile]: Ooops, sorry! 🙇‍♂️ https://github.com/test-unit/test-unit/pull/198/commits/9fdb20d318a9195eed1adbc4294f31d71d5e527e

**Comment by kou** [comment]
Thanks!

**Comment by kachick** [comment]
Thank you!

**Pull request merged**